style.css

```diff
@@ -130,3 +130,416 @@ p {
+  width: 327px;
+  height: 36px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.content-data > p {
+  width: 327px;
+  height: 147px;
+  margin-top: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.content-data > ul {
+  display: flex;
+  justify-content: space-between;
+  gap: 12px;
+  list-style: none;
+  padding-inline-start: inherit;
+}
+
+.list2 {
+  border: 1px solid white;
+  padding: 12px;
+}
+
+.list1 > a {
+  text-decoration: none;
+  padding: 12px;
+  text-align-last: center;
+  border: 1px solid black;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #091e42;
+}
+
+.work1-btn {
+  list-style: none;
+}
+
+.work1-button {
+  padding: 12px;
+  width: 161px;
+  background-color: rgb(255, 107, 0);
+  border: none;
+  color: white;
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-top: 10px;
+}
+
+.work1-btn :hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+}
+
+.work-display {
+  display: grid;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work-2 {
+  width: 327px;
+  margin: auto;
+  margin-top: 24px;
+  background: url(./svg/ImgPlaceholder1.svg);
+  background-color: #1c1a19;
+  background-repeat: no-repeat;
+  background-position-x: center;
+}
+
+.work2-content > * {
+  padding: 0 12px 0 12px;
+}
+
+.work2-content > h2 {
+  margin-top: 80px;
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: white;
+}
+
+.work2-content > h1 {
+  display: none;
+}
+
+.work2-content > p {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  max-width: 100%;
+  max-height: 100%;
+}
+
+.work2-content ul {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 12px;
+  width: 247px;
+  height: 40px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.work2-content li {
+  display: flex;
+  flex-direction: row;
+  align-items: right;
+  padding: 8px 10px;
+  width: 67px;
+  height: 30px;
+  background: rgba(255, 255, 255, 0.24);
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: none;
+}
+
+.work2-content ul li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px;
+  width: 60px;
+  height: 25px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
```

Comment:
- [ ] Please adhere to the *Figma* design and try to implement it as same as possible because as a software developer, it is important to have good detailing skills. Your skills are not looking good or professional to the eyes so kindly fix the issue, which I tried my best to help you to visualize and see the difference.

### Design Comparison

<table>
<tr>
<th align=“middle”>Your Design</th>
<th align=“middle”>Figma Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-31 16-49-35](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/a19a5eaa-be33-4295-a22c-150138d5df61)

</td>
<td>

![Screenshot from 2023-05-31 16-49-43](https://github.com/sulaimanahmadi/Portfolio/assets/66476329/ffe4141e-0368-4bed-a95c-91123ab761a6)

</td>
</tr>
</table>